tests/acceptance/README

```diff
@@ -325,16 +324,30 @@ bundle agent test
 # Activate policy for behaviour you wish to inspect
 {
   meta:
-    "description" string => "What does this test?";
+    "description" -> { "CFE-1234" }
+      string => "What does this test?";
 
     "test_soft_fail"
-      string => "any"
-      meta => { "redmine7803" };
+      string => "Class_Expression|here",
+      meta => { "CFE-1234" };
 }
 
 bundle agent check
 # Check the result of the test
 {
 
+  # Pass/Fail requires a specific format:
+  # reports: "$(this.promise_filename) Pass";
+  # reports: "$(this.promise_filename) FAIL";
+
+  # Consider using one of the dcs bundles
+  # methods: "Pass/Fail" usebundle => dcs_passif( "Namespace_Scoped_Class_Indicating_Test_Passed", $(this.promise_filename) );
+
+}
+bundle agent __main__
+{
+  methods:
+      "Run the default test bundle sequence (init,test,check,cleanup) if defined"
+        usebundle => default( $(this.promise_filename) );
```

Comment:
I would maybe add a comment saying that the testall script actually runs the agent with the acceptance test as the entry point:

```
cf-agent -f ./99_path/to_test.cf
```

(This was not obvious to me for a while).

Reply:
Ack, added, what do you think?